scripts/hf_sim.py

```diff
@@ -402,18 +402,22 @@ def run_hf(local_statfile, n_stat, idx_0, velocity_model=args.velocity_model):
         velocity_name,
         "%s %s %s" % (args.fa_sig1, args.fa_sig2, args.rv_sig1),
         str(args.path_dur),
-        str(head_total + idx_0 * (nt * N_COMP * FLOAT_SIZE)),
         "",
     ]
+    
+    # extra params needed for v6.0
     if utils.compare_versions(args.version, "6.0.3") >= 0:
         hf_sim_args.insert(
```

Comment:
Better to change this back to append?

Reply:
it needs the "' at the end of the list, for a extra newline. 
so unless we removed it from template and then manually add in '\n', we need to insert to the -1 position

Reply:
Ah yeah. Lets just append the "" at the end of the argument building. It makes it less bug prone.